src/main/java/hello/HelloController.java

```diff
@@ -1,16 +1,28 @@
 package hello;
 
-import io.micrometer.core.annotation.Timed;
-import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import javax.servlet.http.HttpServletRequest;
 
 @RestController
-@Timed
-public class HelloController {
-    
+public class HelloController implements Comparable {
+    public static String FOO_VAR = "";
+
     @RequestMapping("/")
-    @Timed
-    public String index() {
+    public String index(HttpServletRequest request) {
         return "Greetings from Spring Boot!";
     }
+
+    public Object clone() {
+        return null;
+    }
+
+    public int compareTo(Object value) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override "equals(Object obj)" to comply with the contract of the "compareTo(T o)" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.35.192.52.128.xip.io/coding_rules#rule_key=squid%3AS1210)